PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/AzureRunProfileState.java

```diff
@@ -100,6 +102,7 @@ protected Operation createOperation() {
 
     protected void onFail(@NotNull Throwable error, @NotNull RunProcessHandler processHandler) {
         onFail(error.getMessage(), processHandler);
+        IntelliJAzureExceptionHandler.getInstance().handleException(project, new AzureToolkitRuntimeException("execute run configuration", error), true);
```

Comment:
seems better to move to the next `onFail`

Reply:
Refactor done